ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -193,7 +196,7 @@ data Mempool m blk idx = Mempool {
                     , TickedLedgerState blk TrackingMK
                     , MempoolSnapshot blk idx
                     )
-             )
+             ) -- FIXME: don't we want to introduce a custom data type for this triple?
```

Comment:
Probably, then the code would be more self-documeting as well

Reply:
On second thought, I think a tuple might be better because avoids one extra level of indirection. Another possibility might be adding the extended and ticked ledger state as a part of the `MempoolSnapshot` type.

Reply:
I'll leave this as-is for the time being, since I'm not completely sure changing the `MempoolSnapshot` type to include theses states is a good idea.